app/views/products/_details_all_content.haml

```diff
@@ -1,9 +1,8 @@
 #detailsPageContent
-  #leftAlign
+  #leftSidebar
     = image_tag @product.picture, id: 'detailsPageImage', class: 'imageWithText'
-    #dietaryRestrictionsFullContent
-      = render 'details_dietary_restrictions'
     #extendTags
       = render 'details_extend_tags'
   #detailsPageText.detailsTextWithImage
-    = render 'details_text'
\ No newline at end of file
+    .pageText
```

Comment:
This class can probably just be appended to the end of the previous line. Unless it is intended to have a div within a div.

Reply:
It's intended to have a div within a div. It makes it a child element, which I needed to lay it out

Layouts are parent-child; you can't do sibling-sibling layouts